src/en/contributing/resources.md

```diff
@@ -10,7 +10,7 @@
 - [Rust JWT](https://github.com/Keats/jsonwebtoken)
 - [Hierarchical tree building javascript](https://stackoverflow.com/a/40732240/1655478)
 - [Hot sorting discussion](https://meta.stackexchange.com/questions/11602/what-formula-should-be-used-to-determine-hot-questions) [2](https://medium.com/hacking-and-gonzo/how-reddit-ranking-algorithms-work-ef111e33d0d9)
-- [Classification types.](https://www.reddit.com/r/ModeratorDuck/wiki/subreddit_classification)
+- [List of subreddit types.](https://www.reddit.com/r/ListOfSubreddits/wiki/listofsubreddits/)
```

Comment:
Is this really relevant? I think it would be good to update this list and categorize it more. Some of these things like the stackoverflow links also seem way too specific to include in documentation. If anyone needs that he can google on his own.

Reply:
I'm not sure who added that one, I don't remember doing so. But some of those other resources should be there. They've been helpful, and its a good place to store them.

I've organized them a bit now too.